games-simulation/flightgear/flightgear-2018.2.2.recipe

```diff
@@ -0,0 +1,118 @@
+COPYRIGHT="2002-2018 Curtis Olsen and others"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/flightgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="61f809ef0a3f6908d156f0c483ed5313d31b5a6ac74761955d0b266751718147"
+PATCHES="flightgear-2018.2.2.patchset"
+ADDITIONAL_FILES="flightgear.rdef"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	flightgear$secondaryArchSuffix
+	cmd:fgcom
+	cmd:fgelev
+	cmd:fgfs
+	cmd:fgjs
+	cmd:fgpanel
+	cmd:fgtraffic
+	cmd:fgviewer
+	cmd:GPSsmooth
+	cmd:JSBSim
+	cmd:js_demo
+	cmd:metar
+	cmd:MIDGsmooth
+	cmd:UGsmooth
+	cmd:yasim
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libglew$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libosg$secondaryArchSuffix = 3.4.1
+	lib:libplibnet$secondaryArchSuffix
+	lib:libqt5$secondaryArchSuffix
+	lib:libsimgearcored$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libglew$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libosg$secondaryArchSuffix = 3.4.1
+	devel:libplibnet$secondaryArchSuffix
+	devel:libqt5$secondaryArchSuffix
+	devel:libsimgearcored$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:cmake >= 3.0
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. \
+		-DFG_DATA_DIR:STRING="`finddir B_SYSTEM_DATA_DIRECTORY`/flightgear" \
+		-DCMAKE_CXX_FLAGS:STRING+=" -lnetwork" -DCMAKE_EXE_LINKER_FLAGS:STRING+=" -lnetwork" \
+		-DCMAKE_BUILD_TYPE:STRING="Release" -DENABLE_PLIB_JOYSTICK:BOOL=OFF \
+		-DWEB_BROWSER:STRING="WebPositive" -DSYSTEM_SQLITE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON \
+		-DENABLE_TESTS:BOOL=OFF
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make -C build install
+	mkdir -p $appsDir
+	mv $binDir/fgfs $appsDir
+	rm -rf $prefix/share
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/flightgear.rdef.in > flightgear.rdef
```

Comment:
This command means: sed have to open the file ```$portDir/additional-files/flightgear.rdef.in```, replace the strings and save as ```flightgear.rdef```.
But it will fail, as your rdef in the additional files folder called as ```flightgear.rdef``` instead of ```flightgear.rdef.in```.
Rename that file.